platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateFileManager.java

```diff
@@ -362,6 +431,7 @@ public void dumpState(
                 reason,
                 success -> latch.countDown(),
                 true,
+                false,
```

Comment:
It may not always be true that the state lacks signatures. I assume this is just to suppress the warning if it is unsigned because, as an out of band state, we don't care if it lacks signatures or not. If this is the case, a comment would be helpful to future readers.

Reply:
In fact, this value isn't even checked for out-of-band writes

I'll add a comment